app/views/posts/show.html.erb

```diff
@@ -10,6 +11,10 @@
         <span> Likes: <%= @post.likes_counter %> </span>
       </div>
     </div>
+    <div class="title-like">
+      <h4>Title:  <%= @post.title %> </h4>
+      <%= button_to "Like", user_post_likes_path(@user.id, @post.id), method: :post %>
+    </div>
     <p class="post-text"> <%= @post.text  %> </p>
     <% if @post.comments.length > 0 %>
       <%= render '/shared/comments', comments: @comments %>
```

Comment:
## OPTIONAL
- [ ] Kindly add a link to the new comment form here

Reply:
Will do so, thanks for reminding.